### PR TITLE
Add documentation about custom DC UI changes

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: custom_dc/customize_ui.md [hunk 1]
```diff
@@ -3,5 +3,92 @@ layout: default
...
+```
+
+in the html with custom content inside the block. More details can be
+found in [Jinja2 template inheritance document](https://jinja.palletsprojects.com/en/3.1.x/templates/#template-inheritance).
+
+### Update CSS and Javascript
+
+Put custom CSS and Javascript files under folder
+[/static/custom_dc/custom](https://github.com/datacommonsorg/website/tree/master/static/custom_dc/custom).
+Then include these files in the `<head>` section of the corresponding html files
```
Comment (beets) [inline custom_dc/customize_ui.md:57]: also mention override.css?

**Review by beets — COMMENTED** [review]
thanks for this. added some comments to make it easier to follow along

File: custom_dc/customize_ui.md [hunk 2]
```diff
@@ -3,5 +3,92 @@ layout: default
...
 parent: Custom Data Commons
-published: false
+published: true
 ---
+
+## Overview
+
+Custom Data Commons allows customization of the web pages on top of
+[datacommons.org](https://datacommons.org). The customization includes overall
+color scheme, home page contents, landing pages of timeline/scatter/map tools
```
[the remark this answers is not in the document]
Comment (shifucun) [reply custom_dc/customize_ui.md]: Done

File: custom_dc/customize_ui.md [hunk 3]
```diff
@@ -3,5 +3,92 @@ layout: default
...
+and etc.
+
+## Environment Setup
+
+Fork [datacommonsorg/website](https://github.com/datacommonsorg/website) Github
+repo following [this
+instruction](https://github.com/datacommonsorg/website#github-workflow) into a
```
[the remark this answers is not in the document]
Comment (shifucun) [reply custom_dc/customize_ui.md]: Done

File: custom_dc/customize_ui.md [hunk 4]
```diff
@@ -3,5 +3,92 @@ layout: default
...
+
+### Update Header, Footer and Page Content
+
+Page header and footer can be customized in
+[base.html](https://github.com/datacommonsorg/website/blob/master/server/templates/custom_dc/custom/base.html)
+by updating the html element within `<header></header>` and `<footer></footer>`.
+Homepage can be customized in
+[homepage.html](https://github.com/datacommonsorg/website/blob/master/server/templates/custom_dc/custom/homepage.html).
+
+Other pages like `about.html` and `faq.html` can be added under
```
[the remark this answers is not in the document]
Comment (shifucun) [reply custom_dc/customize_ui.md]: That's a good idea. I will add those in website and update the doc here.

File: custom_dc/customize_ui.md [hunk 5]
```diff
@@ -3,5 +3,92 @@ layout: default
...
+[/server/templates/custom_dc/custom](https://github.com/datacommonsorg/website/blob/master/server/templates/custom_dc/custom)
+as well. To customize the content in these pages, add the following blocks:
+
+```html
+{% raw %} {% block head %} {% endblock %} {% endraw %}
```
[the remark this answers is not in the document]
Comment (shifucun) [reply custom_dc/customize_ui.md]: Oh, this is only to make jeklly build happy, they don't show up in the rendered markdown

File: custom_dc/customize_ui.md [hunk 6]
```diff
@@ -3,5 +3,92 @@ layout: default
...
+
+Page header and footer can be customized in
+[base.html](https://github.com/datacommonsorg/website/blob/master/server/templates/custom_dc/custom/base.html)
+by updating the html element within `<header></header>` and `<footer></footer>`.
+Homepage can be customized in
+[homepage.html](https://github.com/datacommonsorg/website/blob/master/server/templates/custom_dc/custom/homepage.html).
+
+Other pages like `about.html` and `faq.html` can be added under
+[/server/templates/custom_dc/custom](https://github.com/datacommonsorg/website/blob/master/server/templates/custom_dc/custom)
+as well. To customize the content in these pages, add the following blocks:
```
[the remark this answers is not in the document]
Comment (shifucun) [reply custom_dc/customize_ui.md]: I find it very hard to describe without a real example. So I removed them for now and will add faq.html, about.html and a "new.html" with these blocks in place so I can describe based on that file.

File: custom_dc/customize_ui.md (hunk 1 above)
Comment (shifucun) [reply custom_dc/customize_ui.md:46]: Done

**Review by shifucun — COMMENTED** [review]
thanks for the review!

**Review by beets — APPROVED** [review]
thanks for the updates!

**Pull request merged**